news/2022-12-10-makeship-x-osu-plushies.md

```diff
@@ -0,0 +1,72 @@
+
+Still not convinced? Witness a pippi plushie riding into the (osu! cookie) sunset atop the Taikonator 3000 as she leaves behind a hard life of farming scores for a new adventure that involves actually enjoying the game. She's crying because she's happy.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/yeehaw.jpg)](https://www.makeship.com/search?q=PPY)
+
+Take a moment to look back on how far we've come, from clicking circles and making fan art to actually bringing both of them into the real world. In plushie form, no less.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/the-og.jpg)](https://www.makeship.com/search?q=PPY)
+
+When our dashing developer [smoogipoo](https://osu.ppy.sh/users/1040328) isn't carefully considering code commits, he moonlights as a plushie salesman. Instead of punching time cards, our developers try to sell you things that just straight up punch you instead.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/smoog.jpg)](https://www.makeship.com/search?q=PPY)
+
+Witness the Dean "peppy" Herbert waving his prototype osu! cookie in gentle disapproval at those who dare to leave the template visible in their fan art submissions.
+
+[![](/wiki/shared/news/2022-12-10-makeship-x-osu-plushies/waving-cookie.webp)](https://www.makeship.com/search?q=PPY)
+
+We also felt it was important to demonstrate just how throwable the osu! cookie plushie is. No reason.
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/makeship-x-osu/wang.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you like what you see, make them yours today by helping us fund the Makeship campaigns to bring them into reality. But get in quick, because these plushies are a **limited edition** run - so if you don't buy them this time, they're probably not coming back again!
+
+## How does this all work?
+
+It's real simple, and Makeship also [describes the process on their site](https://www.makeship.com/how-it-works):
+
+- If you want any (or all) of the three plushies above, pledge for them on Makeship by **clicking any of the pictures above**.
+- If we reach our funding goal for a given plushie, the campaign becomes *guaranteed* and everyone who ordered one gets one.
+- Guaranteed campaigns keep taking new orders until they close, so everyone who orders one will get one.
+- If the funding goal isn't reached, all orders for that campaign are refunded, no questions asked.
+
+Basically, buy the plushies and if enough people want one, they get made. If they don't, you get your money back. Easy.
+
+---
+
+**[What're you waiting for? Pledge for your osu!-themed plushies today!](https://www.makeship.com/search?q=PPY)**
+
+Make sure to act quick, since production of these adorable plushies starts on January 1, 2023 - and only for the plushies that meet their funding goal by this date! They're expected to ship out around the end of March 2023, if successful.
```

Comment:
```suggestion
Make sure to act quick, since production of these adorable plushies starts on January 1, 2023 — and only for the plushies that meet their funding goal by this date! They're expected to ship out around the end of March 2023, if successful.
```